await/src/main/java/com/adyen/checkout/await/internal/ui/AwaitDelegate.kt

```diff
@@ -6,14 +6,17 @@
  * Created by oscars on 16/8/2022.
  */
 
-package com.adyen.checkout.await
+package com.adyen.checkout.await.internal.ui
 
+import androidx.annotation.RestrictTo
+import com.adyen.checkout.await.internal.ui.model.AwaitOutputData
 import com.adyen.checkout.components.base.ActionDelegate
 import com.adyen.checkout.components.base.DetailsEmittingDelegate
 import com.adyen.checkout.components.base.StatusPollingDelegate
 import com.adyen.checkout.components.base.ViewableDelegate
 import com.adyen.checkout.components.ui.ViewProvidingDelegate
 
+@RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
```

Comment:
Can this be internal?

Reply:
No as we pass it through the component constructor.